src/http/signals.zig

```diff
@@ -0,0 +1,31 @@
+pub const Signals = struct {
```

Comment:
The file name should be `Signals.zig` and this should be a top-level struct

```suggestion
const Signals = @This();
```